conda/recipes.txt

```diff
@@ -0,0 +1,17 @@
+openjpeg
```

Comment:
Are all of these projects missing from major conda repositories? This is a lot of recipes to maintain.

Reply:
I think most are some of it has to do with wonkiness on the version numbers.  We will try to work to reduce the number of recipes in here.